### DISCUSSION TITLE
Replacement for deprecated `mastodon.social/web/accounts` permalinks?

### DISCUSSION THREAD

**Discussion by beedell-roke** [mastodon/mastodon#22843] [Q&A] [answered]
As [`issues/23009`](https://github.com/mastodon/mastodon/issues/23009#issue-1525505817) describes, I used to be able to access my Mastodon account by utilizing the numerical account identifier hyperlink [`mastodon.social/web/accounts/106986373252061228`](https://mastodon.social/web/accounts/106986373252061228). The provided hyperlink no longer operates.

What replaces it?

**Comment by trwnh** [comment, marked as answer]
nothing, unfortunately. database ids are now masked by acct in the webapp

**Reply by beedell-roke** [reply]
In which case, I've created [`issues/23009`](https://github.com/mastodon/mastodon/issues/23009#issue-1525505817).
